test_rclcpp/CMakeLists.txt

```diff
@@ -131,6 +133,42 @@ if(AMENT_ENABLE_TESTING)
         "rclcpp")
     endif()
 
+    ament_add_gtest(local_parameters_cpp__${middleware_impl} "test/local_parameters.cpp")
+    add_dependencies(local_parameters_cpp__${middleware_impl} ${PROJECT_NAME})
```

Comment:
I just added both to `CMakeLists.txt`, let me know if they look ok and I'll squash and merge this PR.